i18n/lang.json

```diff
@@ -0,0 +1,538 @@
+{
+    "language": "en-US",
+    "chartTitle": "Chart title",
+    "pieSliceName": "Slice",
+    "seriesName": "Series {add series.index 1}",
+    "yAxisTitle": "Values",
+    "rangeSelector": {
+        "monthText": "{count}m",
```

Comment:
Q: Should we also include `week`, `day`, `hour`, `minute` and `second`? Those were not defined in the default buttons of RangeSelector, however, those are quite popular in intraday charts (eg [this](https://www.highcharts.com/demo/stock/intraday-breaks) or [this](https://www.highcharts.com/demo/stock/data-grouping))